scripts/screens/screens_core/screens_core.py

```diff
@@ -68,7 +68,7 @@ def rebuild_core(*, should_rebuild_bgs=True):
     )
     menu_buttons["catlist_screen"] = UISurfaceImageButton(
         ui_scale(pygame.Rect((0, 60), (88, 30))),
-        "screens.core.cat_list",
+        "screens.core.catlist",
```

Comment:
Revert, this will display incorrectly.
```suggestion
        "screens.core.cat_list",
```